### PR TITLE
Kotlin DSL Tooling Builders don't clutter output

### PR DESCRIPTION
**Pull request by eskatos**
Kotlin DSL Tooling Builders do print diagnostic information to stdout.
It ends up in the daemon log files, for diagnostics.
But it also clutters output in IDEs querying for script dependencies during build import.

This PR is about removing the noise in output in IDEs.

### CODE REVIEW

File: subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc [hunk 1]
```diff
@@ -134,6 +134,8 @@ If the above doesn't work and you suspect an issue with the Kotlin DSL script ed
 From version 5.1 onwards, the log directory is cleaned up automatically.
 It is checked periodically (at most every 24 hours) and log files are deleted if they haven’t been used for 7 days.
 
+If the above isn't enough to pinpoint the problem, you can enable the `org.gradle.kotlin.dsl.logging.tapi` system property in your IDE. This will cause the Gradle Daemon to log extra information in its log file located in `&HOME/.gradle/daemons`. In IntelliJ IDEA this can be done by opening `Help > Edit Custom VM Options...` and adding `-Dorg.gradle.kotlin.dsl.logging.tapi=true`.
```
Comment (bamboo) [inline subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc]: ❓ Did you mean `$HOME/.gradle/daemons`?

Comment (eskatos) [reply subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc]: Ooops, typo. Thanks, fixed

Comment (bamboo) [reply subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc]: Should be `$HOME/.gradle/daemon` without the ending `s`, right?

Comment (eskatos) [reply subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc]: arf, Friday double typo ... fixing, thanks

Comment (eskatos) [reply subprojects/docs/src/docs/userguide/api/kotlin_dsl.adoc]: Done, please take another look

**Pull request merged**